src/secrets/CommandList.ts

```diff
@@ -46,13 +46,19 @@ class CommandList extends CommandPolykey {
           logger: this.logger.getChild(PolykeyClient.name),
         });
         const data = await binUtils.retryAuthentication(async (auth) => {
-          const data: Array<string> = [];
+          const data: Array<any> = [];
           const stream = await pkClient.rpcClient.methods.vaultsSecretsList({
             metadata: auth,
```

Comment:
I *really* don't like using the `any` type, can we properly type this?

Reply:
sure

Reply:
i've rebased a commit to change this

Reply:
i think `src/identities/CommandList.ts` might need the same change. Although, I'm a bit caught up on whether the `json` format should be returning the type `{ permissions: Array<...>, nodes: Array<...>, identities: Array<...> }` rather than `GestaltMessage` type. If so, this is a command that is aggregating data from js-rpc, and hence can decide to not follow the js-rpc response type. This would make sense if `identities list` is meant to list identities, rather than gestalts themselves. If not, I'll keep the non-json format output the same, and change the json format to output a similar structure to the rpc response.

Reply:
Not sure, It should only keep the data it needs for for the command. The main thing I want is explicit typing. So no use of `any` type, implicitly or otherwise.

Reply:
I've made it so that that command uses explicit typing, and that the output of the command more closely follows the RPC response.